src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -359,6 +365,27 @@ def get_package_size(self, output):
             return Constants.UNKNOWN_PACKAGE_SIZE
     # endregion
 
+    # region auto OS updates
+    def get_current_auto_os_update_settings(self):
+        """Get current auto OS updates behaviour set on the machine using the command input"""
+        self.composite_logger.log_debug("Fetching current system settings for auto OS updates")
+        try:
+            return self.invoke_package_manager(self.find_current_auto_os_updates_settings_cmd)
+        except Exception as error:
+            self.composite_logger.log_error("Error fetching current system settings for auto OS updates. [Error={0}]".format(str(error)))
+            return None
```

Comment:
Do not assume this on behalf of the system. We should fail the operation in this case